src/Microsoft.ML.EntryPoints/FeatureCombiner.cs

```diff
@@ -312,7 +312,7 @@ public static CommonOutputs.TransformOutput PrepareRegressionLabel(IHostEnvironm
                     }
                 }
             };
```

Comment:
don't see this being used .  can this be deleted ?

Reply:
Yes good point!

---
In reply to: [254010705](https://github.com/dotnet/machinelearning/pull/2365#discussion_r254010705) [](ancestors = 254010705)